src/managers/tls.py

```diff
@@ -177,6 +177,7 @@ def remove_stores(self) -> None:
             self.workload.exec(
                 command=[
                     "rm",
+                    "-rf",
```

Comment:
**thought**: That's funny, I never thought about `rm` return code changing with those flags